vite.config.js

```diff
@@ -4,13 +4,13 @@ import { svelte } from '@sveltejs/vite-plugin-svelte';
 export default defineConfig({
   plugins: [svelte()],
   test: {
+    dir: './',
+    globals: true,
```

Comment:
`dir` is needed so the coverage results are scoped to the files being evaluated by instanbul/v8. If you exclude it, it will include all files in the root directory.